msg/Entity.msg

```diff
@@ -0,0 +1,5 @@
+# Entity ground truth state in the simulation
+
+string name                                 # Entity's scoped, unique name (including namespace)
+string description                          # Optional: additional information about entity.
+EntityState state                           # Entity current state
```

Comment:
I looked at the service definitions below (DeleteEntity, GetEntityState, SetEntityState) and they all take `string name` parameters with the entity name, not an `Entity` message. Is there an advantage to using an `Entity` message without `EntityState` in these service definitions instead of `string name` parameters?

I know we use unique integers to represent entities as an implementation detail in Gazebo, so we could add an optional integer id parameter to `Entity`, but it feels like premature optimization

Reply:
@scpeters noted, it is a good consideration. I think that entityID being and integer is something that Gazebo does, but would not translate well to other simulators. String seems universal as integers can be easily converted to it.
Having said that, I am looking forward to next steps for the standard!

Reply:
I am thinking about cases where we need a ~millions of controllable entities.